src/components/RandomContainer/RandomContainer.tsx

```diff
@@ -14,7 +14,7 @@ type Drink = {
 
 const RandomContainer: FC<RandomDrinks> = ({ randomDrinks }) => {
```

Comment:
I think there might be a way we can refactor line 15  to not even need the `randomDrinks` prop as we have the `Drinks ` Type now. we may be able to bypass this step.